units/host_stats.go

```diff
@@ -0,0 +1,78 @@
+package units
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/model/host"
+	"github.com/mongodb/amboy"
+	"github.com/mongodb/amboy/dependency"
+	"github.com/mongodb/amboy/job"
+	"github.com/mongodb/amboy/registry"
+	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/message"
+)
+
+const hostStatsName = "host-stats"
```

Comment:
what kind of host stats? the job name/scope feels too vauge, and I have to read the code to figure this out.